doc/input-output-reference/src/overview/group-air-distribution-equipment.tex

```diff
@@ -28,6 +28,11 @@ \subsubsection{Inputs}\label{inputs-001}
 
 The design maximum volume flow rate~ (m\(^{3}\)/sec) specified for the AirTerminal:SingleDuct:Uncontrolled object.~ This field is autosizable.
 
+\paragraph{Field: Design Specification Outdoor Air Object Name}\label{field-DSOA-object-name}
+
+This field allows modifying the behavior of this air terminal so that it is modulated to supply the required outdoor air to the zone.  This field is optional. When the name of an DesignSpecification:OutdoorAir object is entered, the model is changed to adjust the flow rate to provide the volume of outdoor air described by that object.  This feature allows modeling demand controlled ventilation on a zone-by-zone basis using the Outdoor Air Flow per Person rate (specified in the DesignSpecification:OutdoorAir object) and the number of occupants (specified in the People object schedules).  If the outdoor air fraction of the supply air is 1.0, as for a dedicated outdoor air system, the air flow rate will match the outdoor air requirement.  When the outdoor air fraction is less than 1.0, as for a recirculating air system, the terminal air flow will be modulated upward to account for the increased total air flow needed to provide the required flow rate of outdoor air.   The total air flow rate will not exceed the Maximum Air Flow Rate specified above. The volume flow rate is converted to mass flow rate using the current density of air (function of drybulb and humidity ratio) at the supply air node.
+
```

Comment:
@lgu1234 @EnergyArchmage Down in the code, I made an earlier comment to use StdRhoAir instead of evaluating the current density, and that change has been made in the code.  The docs need to be changed to match.